site/sfguides/src/vulnerability_intelligence_on_snowflake/vulnerability_intelligence_on_snowflake.md

```diff
@@ -0,0 +1,522 @@
+   ```
+
+## Step 5: Vulnerability Management Use Cases
+
+Duration: 15
+
+### Use Case 1: Assets Impacted by a Headline Vulnerability (CVE Lookup)
+
+The first use case is when there is a new vulnerability disclosure and a CISO is tasked with assessing impact to their environment and has to answer customers if they are impacted. In such a use case, the TIOS Streamlit app can be used to do a CVE ID lookup and find the number of impacted assets.
+
+In the filter form, enter "CVE-2023-0466" into the CVE ID field and apply filter. You will see a number of results. In this case unfortunately we are affected by this vulnerability based on the coverage of our asset scanners.
+
+Similarly, enter "CVE-2024-1234" in to the CVE ID textbox of the filter form and apply filters. You will see there are no results implying that we're not affected by this vulnerability based on the coverage of our asset data.
+
+NOTE: Does not having any results mean we're not impacted by a vulnerability? No! It just means that through the lens our asset data we don't see any assets impacted. The vulnerability could be present in assets that are not currently covered by our asset scanning. This shows the importance of our asset coverage.
+
+### Use Case 2: Vulnerabilities Affecting an Asset (Asset ID Lookup)
+
+For our second use case, let us consider a scenario where we found Indicators of Compromise (IOC) on one of our assets. We now want to lookup the attack surface on that asset as part of the incident response investigation that will also guide our remediation efforts. In this case, TIOS Streamlit app can be used to perform an asset ID lookup.
+
+Paste "i-0o4pkw8dveyp2ijvd" in to the "Asset ID" textbox of the filter form and apply filter. You'll see some results returned which show the unique number of vulnerabilities affecting packages in assets within the organization.
+
+As another example input "i-1234" into the Asset ID textbox of the filter form and apply filter. You'll notice that there no results returned. This means that the asset ID: "i-1234" is not impacted by any vulnerabilities at the moment.
+
+### Use Case 3: Vulnerabilities Affecting a Package (Package Name Lookup)
+
+For our final use case. We find that a headline vulnerability such as log4j emerges and we need to quantify the presence of a package within our environment. In this case TIOS Streamlit app can be used to perform a Package Name lookup.
+
+Paste "openssl" into the "Package Name" textbox of the filter form and apply filter. You'll notice some results.
+
+These results represent vulnerable versions of the package found in various compute instance across the organizations. We can then use this to guide our patching efforts in order to notify each service team of the compute instances in their control.
+
+## Conclusion
+
+Duration: 1
+
+Checkout the blog here: [blog link](). The describes the concepts mentioned in this quickstart in further detail.
```

Comment:
@sfc-gh-pkommini Add the blog link and make this step's layout and structure similar to https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html#8